packages/components/src/components/SquiggleViewer/keyboardNav/unfocusedSqValue.ts

```diff
@@ -0,0 +1,49 @@
+import { SqValuePath } from "@quri/squiggle-lang";
+
+import { toggleCollapsed, useViewerContext } from "../ViewerProvider.js";
+import { focusSqValueHeader, keyboardEventHandler } from "./utils.js";
+
+const validKeys = [
+  "ArrowDown",
+  "ArrowUp",
+  "ArrowLeft",
+  "ArrowRight",
+  "Enter",
+  "e",
+] as const;
+
+export function useUnfocusedSqValueKeyEvent(selected: SqValuePath) {
+  const { setFocused, itemStore, editor, findNode } = useViewerContext();
+
+  return keyboardEventHandler(validKeys, {
+    ArrowDown: () => {
+      const newPath = findNode(selected)?.next()?.node.path;
+      newPath && focusSqValueHeader(newPath, itemStore);
+    },
+    ArrowUp: () => {
+      const newPath = findNode(selected)?.prev()?.node.path;
+      newPath && focusSqValueHeader(newPath, itemStore);
+    },
+    ArrowLeft: () => {
+      const newItem = findNode(selected)?.parent();
+      newItem &&
+        !newItem.isRoot() &&
+        focusSqValueHeader(newItem.node.path, itemStore);
+    },
+    ArrowRight: () => {
+      toggleCollapsed(itemStore, selected);
+    },
+    Enter: () => {
+      setFocused(selected);
```

Comment:
When some other value was already focused, and I try to focus on its subvalue, this doesn't work for some reason.

Reply:
Fixed. Had to add a React ``key`` back in, so it recognized this as a different component than its parent.